examples/hybrid/driver.jl

```diff
@@ -262,7 +262,58 @@ if is_distributed # replace sol.u on the root processor with the global sol.u
         sol = DiffEqBase.sensitivity_solution(sol, global_sol_u, sol.t)
     end
 end
+
+import JSON
+using Test
+import OrderedCollections
 if !is_distributed || (is_distributed && ClimaComms.iamroot(Context))
     ENV["GKSwstype"] = "nul" # avoid displaying plots
     postprocessing(sol, output_dir)
+
+    include(joinpath(@__DIR__, "..", "..", "post_processing", "mse_tables.jl"))
+
+    if !occursin("Float64", job_id) # only do regression tests on Float32 jobs
+
+        Y_last = sol.u[end]
+        # This is helpful for starting up new tables
+        @info "Job-specific MSE table format:"
+        println("all_best_mse[$job_id] = OrderedCollections.OrderedDict()")
+        for prop_chain in Fields.property_chains(Y_last)
+            println("all_best_mse[$job_id][$prop_chain] = 0.0")
+        end
+        @info "Solution variables:"
+        for prop_chain in Fields.property_chains(Y_last)
+            println(prop_chain)
+        end
+
+        # Extract best mse for this job:
+        best_mse = all_best_mse[job_id]
+
+        include(joinpath(
+            @__DIR__,
+            "..",
+            "..",
+            "post_processing",
+            "compute_mse.jl",
+        ))
+
+        ds_computed_filename = joinpath(output_dir, "prog_state.nc")
+
+        export_nc(Y_last, nc_filename = ds_computed_filename)
+        computed_mse = regression_test(
+            job_id,
+            best_mse,
+            ds_computed_filename,
+            ds_computed_filename;
```

Comment:
We’ll need to remove this last positional argument after merging, since this temporarily used to allow merging before NC files have been saved on central.